ts/packages/ts/src/commands/EvalCommand.ts

```diff
@@ -224,6 +227,14 @@ export class EvalCommand extends Command<EvalCommandReturn> {
     return resultWithTemplates;
   }
 
+  private removeForwardsFromPipe(result: EvalCommandReturn): EvalCommandReturn {
+    const { pipe } = this.parameters;
+    if (!pipe) {
```

Comment:
I think this is more accurate, no reason to remove forwards if the user didn't ask for it:
```node.js
    if (!pipe || pipeMode !== EvalCommandPipeMode.Forward) {
      return result;
    }
```